### DISCUSSION TITLE
Most Recent Snapshot Default View - Autoplays Clip when clicking Back to Default

### DISCUSSION THREAD

**Discussion by stevenmccord** [blakeblackshear/frigate#15459] [Ask A Question]
I am not sure if this is a bug, or how it is supposed to work or if I just don't know what I am doing.  But I have my card defaulted to the most recent snapshot.  That works great.  However, when I go to the live view to see the current state and then go back to the default view with the menu button to reset to the default view, it goes to the snapshot and auto plays the clip.  I believe it is because it sees it as an interaction, but I would prefer the default view to just reset and go back to the snapshot image and not autoplay the clip.  The only way I can go back to that is either refreshing the page, or going into the image thumbnails.  Just curious if I am doing something wrong,  or there is some configuration I am not setting.  Appreciate any help!!

**Comment by NickM-27** [comment]
This is not the repo for the Frigate card which seems to be what you're referring to here

**Reply by stevenmccord** [reply]
Oh shoot, sorry about that!
